exporter/dynatraceexporter/go.mod

```
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/armon/go-metrics v0.3.3 // indirect
+	github.com/dynatrace-oss/dynatrace-metric-utils-go v0.1.0
```

```suggestion
	github.com/dynatrace-oss/dynatrace-metric-utils-go v0.2.0
```
Out now, hot and fresh: https://github.com/dynatrace-oss/dynatrace-metric-utils-go/releases/tag/v0.2.0

---

:pie: